src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,57 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.lang.reflect.Array;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_LENGTH = 10;
+    private K[] keyArray;
+    private V[] valueArray;
+    private int firstEmptyIndex;
```

Comment:
I think it would be better
```suggestion
    private int size;
```